lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -351,17 +358,20 @@ Object.defineProperties(
           }
           const routeConfig = { targetData: routeTargetData };
           if (authorizer) {
-            const { name, scopes } = (() => {
+            const { name, scopes, id } = (() => {
               if (_.isObject(authorizer)) return authorizer;
               return { name: authorizer };
             })();
-            if (!authorizers.has(name)) {
+            if (id) {
```

Comment:
I believe we albo should throw, if external authorizer is used with locally configured HTTP API, as that signals error, or some bad practice that may lead to errors.